WorkloadManagementSystem/Executor/JobScheduling.py

```diff
@@ -354,7 +354,9 @@ def __resolveStaging( self, jobState, inputData, idSites ):
     return ( True, bestSites )
 
   def __preRequestStaging( self, jobState, stageSite, opData ):
-    result = getSEsForSite( stageSite )
+    from DIRAC.DataManagementSystem.Utilities.DMSHelpers import DMSHelpers
```

Comment:
You are right actually, since optimizers may not know which VO is concerned, we should get it from the manifest and pass it onto DMSHelpers()